source/aws_signing.c

```diff
@@ -1095,6 +1116,36 @@ static int s_append_canonical_header(
     return AWS_OP_SUCCESS;
 }
 
+/* Add header to stable_header_list to be canonicalized, and also to final signing result */
+static int s_add_authorization_header(
```

Comment:
Called them "authorization headers" because there was a s_add_authorization_query_param() function that does the same thing. Could rename them both s_add_x_amz_headers/query_params() or something.